BootloaderCommonPkg/BootloaderCommonPkg.dec

```diff
@@ -272,16 +272,14 @@
   gPlatformCommonLibTokenSpaceGuid.PcdSourceDebugEnabled      | FALSE      | BOOLEAN | 0x20000212
   gPlatformCommonLibTokenSpaceGuid.PcdContainerBootEnabled    | FALSE      | BOOLEAN | 0x20000213
   gPlatformCommonLibTokenSpaceGuid.PcdEmmcHs400SupportEnabled | TRUE       | BOOLEAN | 0x20000214
-  # Determine if the Pre-OS checker should be executed or not.
-  gPlatformCommonLibTokenSpaceGuid.PcdPreOsCheckerEnabled     | FALSE      | BOOLEAN | 0x20000215
   # This PCD will force to initialize SerialPort regardless of its initialized state
-  gPlatformCommonLibTokenSpaceGuid.PcdForceToInitSerialPort   | FALSE      | BOOLEAN | 0x20000216
+  gPlatformCommonLibTokenSpaceGuid.PcdForceToInitSerialPort   | FALSE      | BOOLEAN | 0x20000215

```

Comment:
why do you change the ID for all other PCDs?
If you just need to remove PcdPreOsCheckerEnabled,  it is not required to change any other PCDs.

Reply:
Noted and uploaded revised patch